cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -83,11 +85,13 @@
     public static final String KEY_RACK = "kafka-rack";
     public static final String KEY_INIT_IMAGE = "init-kafka-image";
     public static final String KEY_AFFINITY = "kafka-affinity";
+    public static final String KEY_KAFKA_LOG_CONFIG = "kafka-log-config";
```

Comment:
wouldn't it be better to name it `kafka-logging`, `zookeeper-logging` and just `logging` for Kafka Connect?